docs/en/writing-running-appium/tutorial/swipe/ios-picker-wheels-mobile.md

```diff
@@ -0,0 +1,78 @@
+## iOS pickerWheels: Slow 'mobile:selectPickerWheelValue' example
+
+When '.setValue()' fails we can use 'mobile:selectPickerWheelValue' way.
+
+This mobile method allows to select next or previous value.
+
+You should change pickerWheel to next/previous and check it new value in
+loop until required appears.
+
+[documentation](../../ios/ios-xctest-mobile-gestures.md#mobile-selectPickerWheelValue)
+
+### Select PickerWheel example
+
+```java
+
+Assert.assertTrue(setPickerWheel("my_text", Order.NEXT), "setPickerWheel(): FAILED");
+
+/**
+ * Set PickerWheel value
+ *
+ * @param text  the text to select
+ * @param order the direction of search
+ * @return result of set
+ * @version java-client: 7.3.0
+ **/
+public boolean setPickerWheel(String text, Order order) {
+    System.out.println("setPickerWheel(): text: '" + text
```

Comment:
Consider using String.format

Reply:
it is better to make 'log' function or use any third-party. but better to keep examples as simple as possible. anyone can make logs in his own way.

Reply:
I mean 

```
String.format("setPickerWheel(): text:'%s'", text)
```

is easier to read and shorter than 
```
String.format("setPickerWheel(): text: '" + text + "'")
```

Reply:
for you easier to read. e.g. for me not. i do no mind to change. i just trying to keep simple.
and if it need to change = it will need to change everywhere.